src/ddmd/mars.d

```diff
@@ -1540,6 +1544,10 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
             {
                 // ignore, already handled above
             }
+            else if (strcmp(p + 1, cast(char*)"cache=mmap") == 0)
```

Comment:
As discussed, should allow some control over which cache file dmd is using.

Reply:
Affirmative, will get to it sometime today / tomorrow.